components/authz-service/storage/postgres/migration/sql/75_update_roles_pols.up.sql

```diff
@@ -0,0 +1,78 @@
+-- replaces "infra:*" with "infra:nodes:*" and "infra:nodeManagers:*"
+-- removes incorrect action "telemetry:*"
+-- removes system permissions
+BEGIN;
+
+UPDATE iam_roles
+    SET
+        actions = '{
+            secrets:*:get,
+            secrets:*:list,
+            infra:nodes:get,
+            infra:nodes:list,
+            infra:nodeManagers:get,
+            infra:nodeManagers:list,
+            compliance:*:get,
+            compliance:*:list,
+            event:*:get,
+            event:*:list,
+            ingest:*:get,
+            ingest:*:list,
+            iam:projects:list,
+            iam:projects:get
+        }'
+    WHERE
+        id = 'viewer';
+
+UPDATE iam_roles
+    SET
+        actions = '{
+            infra:nodes:*,
+            infra:nodeManagers:*,
+            compliance:*,
+            event:*,
+            ingest:*,
+            secrets:*,
+            iam:projects:list,
+            iam:projects:get,
+            iam:projects:assign,
+            applications:*
+        }'
+    WHERE
+        id = 'editor';
+
+UPDATE iam_roles
+    SET
+        actions = '{
+            infra:nodes:*,
+            infra:nodeManagers:*,
+            compliance:*,
+            event:*,
+            ingest:*,
+            secrets:*,
+            iam:projects:list,
+            iam:projects:get,
+            iam:projects:assign,
+            iam:policies:list,
+            iam:policies:get,
+            iam:policyMembers:*,
+            iam:teams:list,
+            iam:teams:get,
+            iam:teamUsers:*,
+            iam:users:get,
+            iam:users:list
```

Comment:
If the docs are valid, project owner is supposed to be a superset of editor, so it also needs  `applications:*`.